src/main/common/filter.c

```diff
@@ -65,6 +65,64 @@ FAST_CODE float pt1FilterApply(pt1Filter_t *filter, float input)
     return filter->state;
 }
 
+// PT2 Low Pass filter
+
+float pt2FilterGain(float f_cut, float dT)
+{
+    float RC = 1 / ( 3.10755f * M_PIf * f_cut); // cutoff 2 * 1.553774 compared to PT1
```

Comment:
The `2 * 1.553774` should be in the code, and not in a comment - there is no point in not putting this into the code, and then needing a comment to explain.

Reply:
OK